tools/src/common/math.h

```diff
@@ -5,7 +5,11 @@
 #ifndef _ACE_TOOLS_COMMON_MATH_H_
 #define _ACE_TOOLS_COMMON_MATH_H_
 
+#if defined(_MSC_VER) && _MSC_VER < 1930
+auto roundToMultipleOf(uint64_t Value, int64_t Multiple) {
```

Comment:
maybe we could use a template here?

```suggestion
template<typename T>
auto roundToMultipleOf(T Value, T Multiple) {
```